src/pages/signin/LoginForm.js

```diff
@@ -151,8 +150,7 @@ class LoginForm extends React.Component {
     }
 
     render() {
-        const formErrorTranslated = this.state.formError && this.props.translate(this.state.formError);
-        const error = formErrorTranslated || ErrorUtils.getLatestErrorMessage(this.props.account);
+        const error = this.state.formError && this.props.translate(this.state.formError);
```

Comment:
Perhaps I'm wrong, but it seems like this change means we are no longer checking for errors from `this.props.account`. Is that correct?

Reply:
> I guess you're correct since we no longer need this method

To clarify, I think we still need to check for errors in `this.props.account`. We don't need the method because if there are many errors, we will show all of them (instead of just the first one).

Reply:
Let's get @bondydaa 's opinion

Reply:
You're right @Luke9389 , showing all the errors now

Reply:
@bondydaa How should we display multiple errors when the FormAlertWithSubmitButton component only takes one error message from the LoginForm since the RBR won't be available here? 🤔

Reply:
I think we should probably get rid of the internal state of this component and rely solely on if there are errors coming back from onyx. 

update `error` so that it's pulling the data from onyx which should be an array or object of potential errors, pass that directly to the components then.

Then update the components that we're passing the `error` to so that it iterates over the `error`s passed and displays them via the brick road component that I think Jasper created.

Reply:
So we expect to see multiple errors on the LoginForm if they exist @bondydaa?

Reply:
why wouldn't we display all the errors thrown?

Reply:
It's most likely that these commands can really only throw 1 error but seems strange we'd limit this to just one error in the event multiple were thrown and I believe with Brick Road handling the idea is we'll show all errors to the user in a nice way.

Reply:
Ok so for @Justicea83's sake here, which component are we using to display all the errors? @jasperhuangg do you know off-hand?

Reply:
this is component https://github.com/Expensify/App/pull/10556/files#diff-ed15e2a2b3ba051db0688c6e465dae1d08147a3dabaa4bead5cb6b7f9c361f76 `DotIndicatorMessage`

Reply:
Awesome! Ok I think we've set you up for success @Justicea83

Reply:
Thanks @bondydaa 🙇